crypto/init.c

```diff
@@ -37,6 +37,10 @@
 static int stopped = 0;
 static uint64_t optsdone = 0;
 
+/* needs to be global so it is visible to dllmain.c on Windows */
+extern int ossl_no_cleanup;	/* workarounds CI failures on github */
+int ossl_no_cleanup = 0;
```

Comment:
```suggestion
#ifdef OPENSSL_NO_ATEXIT
int ossl_no_cleanup = 1;
#else
int ossl_no_cleanup = 0;
#endif
```

And revert the changes in Configure and mention the new meaning of no-atexit in the CHANGES.md entry.

Reply:
yes, it makes sense. thanks.